tests/cli/test_main_config.py

```diff
@@ -37,6 +37,106 @@
     )
 
 
+@pytest.mark.integration
+def test_env_create_with_channels_from_yaml(
```

Comment:
I think these tests would fit better in `tests/cli/test_env.py`

Reply:
I struggled with where to put them.  I can move them there.